src/filter.js

```diff
@@ -0,0 +1,241 @@
+
+    const filterText = document.createElement("div");
+    filterText.innerHTML = text;
+    filterText.style.color = "rgb(25,25,25)";
+    filterText.style.fontFamily = "IBM Plex Sans";
+    filterText.style.fontStyle = "normal";
+    filterText.style.fontWeight = "700";
+    filterText.style.fontSize = "16px";
+    filterText.style.lineHeight = "21px";
+    filterText.style.marginLeft = "25px";
+    return filterText;
+}
+
+function filterButtonRowUi(buttons) {
+    const filterUI = document.createElement("div");
+    filterUI.style.display = "flex";
+    filterUI.style.justifyContent = "center";
+    for (let button of buttons) {
+        const fButton = filterButton(button);
+        filterUI.appendChild(fButton);
+    }
+    return filterUI;
+}
+
+function filterButton(text) {
+    // Button
+
+    const filterButton = document.createElement("div");
+
+    filterButton.innerHTML = text;
+    filterButton.style.background = "#F2F2F2";
+    filterButton.style.color = "#333333";
+    filterButton.style.borderRadius = "100px";
+    filterButton.style.width = "80px";
+    filterButton.style.height = "32px";
+    filterButton.style.marginBottom = "20px";
+    filterButton.style.marginTop = "6px";
+    filterButton.style.marginLeft = "10px";
+    filterButton.style.display = "flex";
+    filterButton.style.flexDirection = "row";
+    filterButton.style.justifyContent = "center";
+    filterButton.style.alignItems = "center";
+    filterButton.style.fontFamily = "Futura PT";
+
+    filterButton.addEventListener("click", () => {
+        const words = text.split(' ');
+        buttonValue[words[words.length - 1]] = !buttonValue[words[words.length - 1]];
+        if (buttonValue[words[words.length - 1]]) {
+            filterButton.style.background = "#E6F4FD";
+            filterButton.style.color = "#2F97D1";
+        } else {
+            filterButton.style.background = "#F2F2F2";
+            filterButton.style.color = "#333333";
+        }
+    });
+    return filterButton;
+}
+
+
+function filterApply(text) {
+    const ApplyButton = document.createElement("div");
+    ApplyButton.innerHTML = text;
+    ApplyButton.style.textAlign = "center";
+    ApplyButton.style.backgroundColor = "#FFFFFF";
+    ApplyButton.style.border = "1px solid #77CCFF";
+    ApplyButton.style.borderRadius = "100px";
+    ApplyButton.style.marginTop = "5px";
+    ApplyButton.style.marginLeft = "10px";
+    ApplyButton.style.fontFamily = "Futura PT";
+    ApplyButton.style.fontStyle = "normal";
+    ApplyButton.style.color = "#2F97D1";
+    ApplyButton.style.width = "100px";
+    ApplyButton.style.height = "32px";
+    ApplyButton.style.position = "absolute";
+    ApplyButton.style.top = "89%";
+    ApplyButton.style.left = "38%";
+
+    ApplyButton.addEventListener("click", () => {
+        const keys = Object.keys(buttonValue);
+        const values = keys.filter(function (key) {
+            return buttonValue[key]
+        });
+        console.log(values);
```

Comment:
Update the Taps state hook when clicking on the button apply

Reply:
The tags state hook now changed when the apply button is triggered ([bd8e2d2](https://github.com/mymizu/mymizu-web/pull/21/commits/bd8e2d2f3de1620993e85b003374cd94c8fa85b2))